### ISSUE TITLE
Tracking validation loss? 

### ISSUE THREAD

**Issue by rlenain** [SWivid/F5-TTS#218]
Hello, and thanks a lot for open sourcing this work.

I am using the `finetune-cli.py` script, which works great! I'm only able to track the training loss, and don't see validation loss being calculated anywhere? Is this supported at all? Or is there a reason why it's not?

Thanks!

**Comment by SWivid** [comment]
No specific reason lol, we just had not organized a val set.
We basically check by generating few samples to hear during training #9

**Comment by rlenain** [comment]
okay that makes sense! thanks

**Comment by SWivid** [comment]
will close this issue, feel free to open if further questions~

**Comment by mukherjeesougata-eros** [comment]
> [@rlenain](https://github.com/rlenain) this script **finetune-cly.py** make for the **gradio finetune** so please change your value you need do not use default something good working last test i make it's
> 
> ```
> learning_rate = 1e-5 
> 
> batch_size_per_gpu = 3200 # for 3090 and 4090 if you have other card see memory ! you need change lower 
> batch_size_type = "frame"  # "frame" or "sample"
> max_samples = 64  # max sequences per batch if use frame-wise batch_size. we set 32 for small models, 64 for base models
> grad_accumulation_steps = 1  # note: updates = steps / grad_accumulation_steps
> max_grad_norm = 1.
> 
> epochs = 100  # i use 100 to stop manual the train this not mean you need do 100 epochs stop when you think it's good
> num_warmup_updates = 300 # warmup steps for this for 4500 samples about
> save_per_updates = 2000  # save checkpoint per steps
> last_per_steps = 10000 # save last checkpoint per steps 
> ```
> 
> i try find some good values i change soon

Can I use the same setup for finetuning F5TTS Small model using `train.py` as given in #57
